src/billing/types/enums/reasonCodeTranscript.ts

```diff
@@ -0,0 +1,163 @@
+import { ReasonCode } from './reasonCode';
+
+/**
+ * Transcript of transaction rejection code for payment failed event
+ * Transcript will be used in notification after words "because of"
+ * https://developers.cloudpayments.ru/#kody-oshibok
+ */
+export const ReasonCodesTranscript = {
```

Comment:
Мб сделать все ошибки с большой буквы?

Reply:
Будет странно смотреться, к примеру: "because of The card is not available for online payments". Если где-то еще будет использоваться, то лучше js-ом заглавной первую букву делать, чем строчной.

Reply:
👌